h/static/styles/annotations.scss

```diff
@@ -71,6 +78,15 @@
   }
 }
 
+.annotation-body {
```

Comment:
These styles should probably all be handled by the excerpt directive and it's corresponding classes otherwise it becomes entangled with the annotation.

Writing the `excerpt` as a directive makes the most sense if it can be isolated. If the annotation needs its own styles in order to interact with it things look less like modular components and more like spaghetti that spreads out across multiple files and modules that are not, in fact, really separate.

There are certainly occasions where it makes sense to have contextual special cases for style interactions, but if that's the case I recommend getting the excerpt directive to control semantically meaningful class names and then use combinations of selectors in the CSS to make the annotation behave accordingly.

tl;dr `show-full-body` is explicitly about the annotation body, and is much too narrow. It means something has to be aware of the interaction with the more/less excerpts _and_ understand the context in which its used (the annotation body) in order to toggle the appropriate classes. Much better would be to have `.excerpt-expanded .annotation-body` or even define things the inverse way by saying what `.excerpt .annotation-body` looks like and causing the `excerpt` class to be removed.